lang/scala3.0/Portfile

```diff
@@ -56,3 +56,5 @@ destroot {
         ln -s ../share/scala-${branch}/bin/${f} ${destroot}${prefix}/bin/${f}-${branch}
     }
 }
+
+github.livecheck.regex {(3\.0\.\d+)}
```

Comment:
Just my thoughts: wouldn't you want to keep the livecheck to find newer versions than 3.0.x - how else would you know there is an updated version? Once there is a 3.1 release and you make a `scala3.1` port, you could restrict it to the 3.0.x branch here. 

If the purpose right now is to not see release candidates, you could use the conventional `github.livecheck.regex  {([0-9.]+)}` to accomplish that.

Reply:
@reneeotten good idea, thanks!